cmd/qat_plugin/qat_plugin.go

```diff
@@ -103,9 +102,10 @@ func (dp *devicePlugin) getDpdkDevice(id string) (string, error) {
 		if err != nil {
 			return "", errors.WithStack(err)
 		}
-		s := strings.TrimPrefix(group, sysfsIommuGroupPrefix)
-		fmt.Printf("The vfio device group detected is %v\n", s)
-		return s, nil
+		s := strings.Split(group, sysfsIommuGroupPrefix)
+
+		fmt.Printf("The vfio device group detected is %v\n", s[len(s)-1])
+		return s[len(s)-1], nil
```

Comment:
Can't we use just `path.Base(group)` here instead of the string manipulations?

Reply:
Cool, this is a good way of implementing what I intending to do!! Thanks for directing me to this

Reply:
Updated!!